File: Baekjoon/Class1/33.2920 음계.py
import sys
input = sys.stdin.readline

is_ascending = True
is_descending = True
notes = list(map(int, input().split()))
prev = notes[0]
for cur in notes[1:]:
    if prev >= cur: is_ascending = False
    if prev <= cur: is_descending = False
    prev = cur
if(is_ascending): print("ascending")
elif(is_descending): print("descending")
else: print("mixed")

File: Baekjoon/Class1/33.2920 음계.js
const inputFile = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const input = require("fs").readFileSync(inputFile, "utf8").trim();

const notes = input.split(" ").map(Number);
let isAscending = true, isDescending = true;
let prev = notes[0];
for(let i = 1; i < notes.length; i++) {
    const cur = notes[i];
    if(prev >= cur) isAscending = false;
    if(prev <= cur) isDescending = false;
    prev = cur;
}
if(isAscending) console.log("ascending");
else if(isDescending) console.log("descending");
else console.log("mixed");